src/CmsAdmin/CategoryController.php

```diff
@@ -215,13 +231,15 @@ public function editAction(Request $request)
      */
     public function moveAction(Request $request)
     {
-        if (null === $category = (new CmsCategoryQuery)->findPk($request->id)) {
+        if (null === $category = (new CmsCategoryQuery)
```

Comment:
podobnie jak już gdzieś wyżej - 
1) zmienna jest potem używana, więc zadeklarowanie jej w `if`'ie nieco zamydla obraz tego co się w tej metodzie dzieje 
2) na pierwszy rzut oka nawet nie widać gdzie się kończy warunek a zaczyna to, co po jego spełnieniu, bo trzeba szukać nawiasów